### ISSUE TITLE
Improving quality assurance and development efficiency with regression testing (use case digital edition)

### ISSUE THREAD

**Issue by babslgam** [DARIAH-ERIC/dariah-campus#1283]
### Title of the resource

Improving quality assurance and development efficiency with regression testing (use case digital edition)

### Resource type

Hosted Resource

### Authors, editors and contributors

Barbara Krautgartner

### Topics (keywords)

Software development

### Learning outcomes

Understand how Playwright can be used for regression testing

### Abstract

This resource delivers an insight into the advantages of using e2e tests through a specific use case.
It provides examples of how the testing tool Playwright is utilized during the migration process of a digital edition.

**Comment by VickyGarnett** [comment]
Hi @babslgam - I've added your name to the system, so you can go ahead with drafting this resource up.  Some pointers to consider, though: have another look at the learning outcomes, I think there are perhaps a few smaller steps in here that can be given as learning outcomes from this resource around the understanding of some of the fundamental concepts behind the software, and how it can be applied in practice.  You should probably aim for around 3 to give learners a clear understanding of what the course will cover.

I also usually have a quick call with new contributors to Campus to run them through the whole process and to point out the common pitfalls that can come up.  If you have some time in the coming weeks, we should set that up.  Please feel free to email me.  Thanks - V

**Comment by VickyGarnett** [comment]
Hi @babslgam - do you want to drop me an email so we can arrange a call to go through this resource?  Thanks :) V

**Comment by VickyGarnett** [comment]
Hi @babslgam - please take a look at the notice on https://github.com/DARIAH-ERIC/dariah-campus/issues/1373 as it contains important information about your resources and the switch to the new Content Management System.  Many thanks!  Vicky

**Comment by babslgam** [comment]
hi @VickyGarnett, thanks for info! i will use the 4 weeks to give my resource the final touch.

**Comment by VickyGarnett** [comment]
Hey @babslgam - I know you're flat out, but have you been able to make any progress on this resource?

**Comment by VickyGarnett** [comment]
Expected Completion Date updated to 31 Dec in agreement with the author

**Comment by VickyGarnett** [comment]
Hi @babslgam - just checking in to see where you are with this?

**Comment by babslgam** [comment]
hi @VickyGarnett, sorry for the delay.
i think version 1.1.0 of the resource is ready for editorial feedback:
https://dariah-campus-git-content-regression-testing-for-0fd1ee-dariah.vercel.app/resources/hosted/improving-quality-assurance-and-development-efficiency-with-regression-testing
thanks!

**Comment by VickyGarnett** [comment]
Thanks @babslgam - is this this on a pull request somewhere?  Thanks :)

**Comment by babslgam** [comment]
yes, pr is here: https://github.com/DARIAH-ERIC/dariah-campus/pull/1667
